data_steward/tools/generate_ehr_upload_pids.py

```diff
@@ -15,76 +17,88 @@
 LOGGER.addHandler(handler)
 LOGGER.setLevel(logging.DEBUG)
 
-EHR_UPLOAD_PIDS_BQ_SCRIPT = JINJA_ENV.from_string('''
-SELECT ARRAY_TO_STRING(ARRAY_AGG(FORMAT
-("""SELECT 
-  person_id,
-  current_datetime() AS report_run_time,
-  Org_ID as org_id,
-  HPO_ID as hpo_id,
-  Site_Name as site_name, 
-  TIMESTAMP_MICROS(t.last_modified_time * 1000) AS latest_upload_time
+pid_tables = [
+    PERSON, CONDITION_OCCURRENCE, PROCEDURE_OCCURRENCE, DRUG_EXPOSURE,
+    OBSERVATION, VISIT_OCCURRENCE
+]
+
+HPO_IDS_QUERY = JINJA_ENV.from_string("""
+SELECT LOWER(hpo_id) AS hpo_id FROM `{{project_id}}.{{dataset_id}}.{{table_id}}`
+""")
+
+EHR_UPLOAD_PIDS_BQ_QUERY = JINJA_ENV.from_string("""
+CREATE OR REPLACE VIEW `{{project_id}}.{{operations_dataset_id}}.{{ehr_pids_view}}` 
+OPTIONS(description='A participant-level view of when ehr data was sent. NOTE: the RDR calls this view to support HealthPro (1/27/21)')
+AS {% for hpo_site in hpo_sites %}
+SELECT 
+    person_id,
+    current_datetime() AS report_run_time,
+    Org_ID as org_id,
+    HPO_ID as hpo_id,
+    Site_Name as site_name, 
+    TIMESTAMP_MICROS(t.last_modified_time * 1000) AS latest_upload_time
```

Comment:
@hiro-mishima Our biggest problem with the ehr_upload_pids was the "Query too large" error, which prevented us from doing this. This feels like it is making the query more complicated. Let me know if you've already tested this and it works fine.